srcpkgs/synapse/template

```diff
@@ -34,10 +39,15 @@ make_dirs="
 
 pre_check() {
 	unzip ${make_install_target} "*.dist-info/*"
+	cp -r tests "build/lib.linux-${XBPS_MACHINE/-*/}-cpython-${py3_ver/./}"
```

Comment:
```suggestion
	[ -d build/lib* ] && cp -r tests build/lib*
```
will avoid hard-to-read shell substitutions and  should trigger a failure if more than one build/lib* matches the glob.